bin/retrieve-widget-js-file.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/bash
```

Comment:
This script retrieve only the widget ?

I expect a script that retrieve all files needed to make a widget that works properly (css, fonts, types, map, etc).

Reply:
ok it is done